EIPS/eip-2535.md

```diff
@@ -15,7 +15,7 @@ requires: 1538, 2535
 ## Abstract
 <img align="right" src="../assets/eip-2535/diamond.svg" width="230" height="230">
 
-This EIP proposes a standard for creating modular smart contract systems that can be extended after deployment ("Diamonds"). This enables smart contracts to effectively have no size limit, and to be upgraded without having to redeploy existing functionality. A Diamond's "Facets" can be added, replaced, or removed without having to worry about unintended side-effects, providing a layer of abstraction that aids development. 
+This EIP proposes a standard for creating modular smart contract systems that can be extended after deployment ("Diamonds"). This enables smart contracts to effectively have no size limit, and to be upgraded without having to redeploy existing functionality. A Diamond's functions can be added, replaced, or removed individually, providing a layer of abstraction that aids development. 
```

Comment:
Yes, this is better phrasing.  I think it is a good idea to just remove this line:   `A Diamond's functions can be added, replaced, or removed individually, providing a layer of abstraction that aids development.`

Reply:
I've changed the phrasing. Is the new version better?